client/src/entrypoints/admin/core.js

```diff
@@ -299,8 +299,8 @@ $(() => {
   const showTab = (tabButtonElem) => {
     $(tabButtonElem).tab('show');
 
-    // Update data-current-tab attribe on the .tab-nav element
-    const tabNavElem = tabButtonElem.closest('.tab-nav');
+    // Update data-current-tab attribute on the [data-tab-nav] element
+    const tabNavElem = tabButtonElem.closest('[data-tab-nav]');
```

Comment:
Are we OK to make this breaking change? (ping @gasman @thibaudcolas) I think there are third-party apps that use the tabs, but this is undocumented

Reply:
Could we change the selector to something like `'ul.tab-nav .tab-nav > ul'` so It caters for both?

Reply:
Is this a big issue? I'd thought we generally tried to avoid attaching too much js using pure css class selectors, so I thought changing to a data attribute was clearer. Also quite easy to change any external dependencies, just change the selector.

But happy to change our css selector instead of using the data attribute if that's easier?